src/server/api/controllers/vessel-reports.controller.js

```diff
@@ -74,8 +74,28 @@ const getVesselReportById = async id => {
   }
 };
 
+//Get Suggested-route by vessel-report-id
+const getsuggestedRouteByVesselReportId = async id => {
+  try {
+    const suggestedRoutes = await knex("suggested_routes")
+      .select("*")
+      .where({ vessel_report_id: id });
+    if (suggestedRoutes.length === 0) {
+      throw new HttpError(
+        "Bad request",
+        `Cannot find any Suggested route for Vessel-report ID ${id}!`,
+        404
+      );
+    }
+    return suggestedRoutes;
+  } catch (err) {
+    return err.message;
+  }
+};
+
 module.exports = {
   createVesselReport,
   getVesselReportById,
-  getVesselsReportByVoyageId
+  getVesselsReportByVoyageId,
+  getsuggestedRouteByVesselReportId
```

Comment:
Can you rename the function name to "getSuggestedRouteByVesselReportId"

Reply:
changed